lib/dndquote.sty

```diff
@@ -9,16 +10,15 @@
 	boxsep=0.25ex,
 	left=8pt,
 	right=8pt,
-	colback=bgtan,
-	colframe=bgtan,
+	colback=quoteboxcolor,
```

Comment:
I had originally suggested adding the color override for commentbox, dndtable, paperbox, and quotebox partially because I could not see a way to change the color for these environments at all. The suggestion of a global color for each of the environments fixed most of the issue, and the one color change was kept as well. I had a bit of an, "Oh, wait a minute," moment a couple days ago when I realized that we could always change the colors through the first optional argument.

Further digging had me realize that the "colframe" parameter is not being used because of the "frame hidden" parameter. So I stripped colframe out. I then thought about removing the optional color parameter for the following reasons:
1. Depending on which tcolorbox is being used there are now only 1 or 2 color parameters that need to be set for a one-off color change. 
2. I am assuming that one-off changes to color will be used very rarely now that there are the global colors to use.
3, There already was a means of changing the environment's color that I had originally missed.

I was not able to figure out how to remove the optional color parameter from dndtable (or rather inline it with the column definition parameter. So I left it as is. The part that I do not like is that you have to call `\begin{dndtable}[][color]` if you want the default XX columns with a one-off color change.